### ISSUE TITLE
All rows in edit mode?

### ISSUE THREAD

**Issue by djlaney3** [stsrki/Blazorise#2352]
Is it possible to have all the rows editable at once?  I am currently using blazorise 0.9.3.5 through the abp.io framework.

**Comment by stsrki** [comment]
No, the editing of all columns at once is not supported. The only way for you to do it currently is to implement it from scratch by using the Table component(s).

There were some ideas already for this feature but the time was always limited. I will put it on the backlog for one of the next milestones.

**Comment by stsrki** [comment]
Not maybe what is requested, but there is now a cell edit mode, which might work as an alternative.

Closing now.
